### PR TITLE
[enhance](mtmv)refresh hms table before run mtmv task

### PR DESCRIPTION
**Pull request by zddr**
There are some caches in the HMS catalog, which can result in the inability to maintain consistency with Hive side data even after a successful refresh of the materialized view.
In this PR, the cache of relevant HMS tables will be cleared before refreshing the materialized view to solve this problem

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

File: regression-test/suites/mtmv_p0/test_hive_refresh_mtmv.groovy [hunk 1]
```diff
@@ -157,9 +148,6 @@ suite("test_hive_refresh_mtmv", "p0,external,hive,external_docker,external_docke
                                     """
```
Comment (seawinde) [inline regression-test/suites/mtmv_p0/test_hive_refresh_mtmv.groovy:148]: after refresh materialized view, maybe we should add `order_qt` to check the mv result is right or not

**Review by morrySnow — APPROVED** [review]

**Review by seawinde — APPROVED** [review]

**Pull request merged**